### PR TITLE
fix: multiple workspaces compatibility list

### PR DESCRIPTION
**Pull request by themr0c**
## What does this pull request change?

Add a compatibility list table in Configuring the number of workspaces that a user can run

## What issues does this pull request fix or reference?

Docs was missing mention of EBS.
The list was repoirted to be confusing.

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc [hunk 1]
```diff
@@ -8,12 +8,22 @@ This procedure describes how to configure `{prod-short}` to run more than one wo
 * You have installed an instance of `{prod-short}` by using Helm and the default namespace `{prod-namespace}`.
 * The combination of PVC strategy and access mode meets the following criteria:
```
Comment (themr0c) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]:
```suggestion
* The combination of PVC strategy and access mode matches one of the following cases:
```

File: modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc [hunk 2]
```diff
@@ -8,12 +8,22 @@ This procedure describes how to configure `{prod-short}` to run more than one wo
 * You have installed an instance of `{prod-short}` by using the Operator.
 * The combination of PVC strategy and access mode meets the following criteria:
```
Comment (themr0c) [inline modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]:
```suggestion
* The combination of PVC strategy and access mode matches one of the following cases:
```

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc [hunk 3]
```diff
@@ -6,14 +6,24 @@ This procedure describes how to configure `{prod-short}` to run more than one wo
 .Prerequisites
 
 * You have installed an instance of `{prod-short}` by using Helm and the default namespace `{prod-namespace}`.
-* The combination of PVC strategy and access mode meets the following criteria:
+* The combination of PVC strategy and access mode matches one of the following cases:
```
Comment (rolfedh) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]:
```suggestion
* The combination of persistent volume claim (PVC) strategy and access mode matches one of the following cases:
```
Consider spelling out this initialism unless you're certain that it is familiar to the reader.

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc [hunk 4]
```diff
@@ -6,14 +6,24 @@ This procedure describes how to configure `{prod-short}` to run more than one wo
...
+| `per-workspace` | yes | yes
+| `unique` | yes | yes
+|===
 +
-See xref:configuring-storage-strategies.adoc[].
+** To determine the access modes supported by your storage, see link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[Kubernetes documentation - Access Modes] 
+** To determine the PVC strategy, see xref:configuring-storage-strategies.adoc[]. 
++
+.Preparing to run multiple workspaces on Amazon Elastic Block Store (Amazon EBS)
+====
```
Comment (rolfedh) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]:
```suggestion
IMPORTANT
====
```

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc [hunk 5]
```diff
@@ -6,14 +6,24 @@ This procedure describes how to configure `{prod-short}` to run more than one wo
...
+link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html[Amazon EBS] only supports the `ReadWriteOnce` access mode. Use the `per-workspace` or `unique` PVC strategy.
```
Comment (rolfedh) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]:
```suggestion
link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html[Amazon EBS] supports only the `ReadWriteOnce` access mode. Use the `per-workspace` or `unique` PVC strategy.
```

Comment (rolfedh) [reply modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: https://www.quickanddirtytips.com/education/grammar/%E2%80%9Conly%E2%80%9D-the-most-insidious-misplaced-modifier

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc (hunk 4 above)
Comment (themr0c) [reply modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: To me, IMPORTANT is meaningless here, it is just an example.

File: modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc [hunk 6]
```diff
@@ -6,14 +6,24 @@ This procedure describes how to configure `{prod-short}` to run more than one wo
 
 .Prerequisites
 * You have installed an instance of `{prod-short}` by using the Operator.
-* The combination of PVC strategy and access mode meets the following criteria:
+* The combination of PVC strategy and access mode matches one of the following cases:
```
Comment (rolfedh) [inline modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]:
```suggestion
* The combination of persistent volume claim (PVC) strategy and access mode matches one of the following cases:
```

File: modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc [hunk 7]
```diff
@@ -6,14 +6,24 @@ This procedure describes how to configure `{prod-short}` to run more than one wo
...
+| `per-workspace` | yes | yes
+| `unique` | yes | yes
+|===
 +
-See xref:configuring-storage-strategies.adoc[].
+** To determine the access modes supported by your storage, see link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[Kubernetes documentation - Access Modes] 
+** To determine the PVC strategy, see xref:configuring-storage-strategies.adoc[]. 
++
+.Preparing to run multiple workspaces on Amazon Elastic Block Store (Amazon EBS)
+====
```
Comment (rolfedh) [inline modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]:
```suggestion
IMPORTANT
====
```

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc (hunk 4 above)
Comment (rolfedh) [reply modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: https://redhat-documentation.github.io/supplementary-style-guide/#admonitions Is this an admonition block? If so, consider using one of the valid admonition types. If you don't want to use one of the valid admonition types, you can also remove the block markers and have the phrase as a single sentence paragraph.

Comment (rolfedh) [reply modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: It seems like an important piece of information that the user must know to choose the correct PVC strategy.

Comment (themr0c) [reply modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: It is an example, not an admonition block.

Comment (rolfedh) [reply modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: I see now how this looks like [example block syntax](https://docs.asciidoctor.org/asciidoc/latest/blocks/example-blocks/). But the content is not an example. It states a limitation and gives the user an instruction.

**Review by rolfedh — COMMENTED** [review]
This PR contains [an issue](https://github.com/eclipse/che-docs/pull/2147/files#r738277453) that should probably be resolved before we merge it.

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc [hunk 8]
```diff
@@ -6,14 +6,24 @@ This procedure describes how to configure `{prod-short}` to run more than one wo
...
+.Multiple workspaces compatibility list
+|===
+| PVC Strategy | `ReadWriteMany` access mode | `ReadWriteOnce` access mode
+| `common` (default) | yes | no
+| `per-workspace` | yes | yes
+| `unique` | yes | yes
+|===
 +
-See xref:configuring-storage-strategies.adoc[].
+** To determine the access modes supported by your storage, see link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[Kubernetes documentation - Access Modes]
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]:
```suggestion
** To determine the access modes supported by your storage, see link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[Kubernetes documentation - Access Modes].
```

File: modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc [hunk 9]
```diff
@@ -6,14 +6,24 @@ This procedure describes how to configure `{prod-short}` to run more than one wo
...
+.Multiple workspaces compatibility list
+|===
+| PVC Strategy | `ReadWriteMany` access mode | `ReadWriteOnce` access mode
+| `common` (default) | yes | no
+| `per-workspace` | yes | yes
+| `unique` | yes | yes
+|===
 +
-See xref:configuring-storage-strategies.adoc[].
+** To determine the access modes supported by your storage, see link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[Kubernetes documentation - Access Modes]
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]:
```suggestion
** To determine the access modes supported by your storage, see link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[Kubernetes documentation - Access Modes].
```

**Review by rkratky — APPROVED** [review]

**Comment by themr0c** [comment]
@rolfedh removed the block causing trouble. Inline example instead.

**Review by ibuziuk — COMMENTED** [review]
Could you clarify if this PR is a replacement of https://github.com/eclipse/che-docs/pull/2146
or a completely separate one? Basically, from my side the requirement is to re-store the warning from https://github.com/eclipse/che-docs/pull/1902

**Comment by themr0c** [comment]
@ibuziuk It's a separate one. I tried to keep the scope of each pull request as small as possible to make review easier. It didn't work as expected, it is causing more confusion./

**Review by ibuziuk — APPROVED** [review]

**Pull request merged**